packages/cascara/src/modules/DataImage/DataImage.fixture.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import DataImage from './DataImage';
+import ModuleSandbox from '../ModuleSandbox';
+
+const DisplayDataImage = ({ isEditing, value, ...rest }) => (
+  <ModuleSandbox isEditing={isEditing}>
+    <DataImage {...rest} label='My button' />
+  </ModuleSandbox>
+);
+
+const DataDateModule = () => {
+  return (
+    <>
+      <div>
+        <h1>Date input field</h1>
```

Comment:
I think this should be `Image input field`

```suggestion
        <h1>Image input field</h1>
```

Reply:
New fixture file